canvas/migrations/0009_auto_20210702_1305.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.14 on 2021-07-02 20:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('canvas', '0008_event_type'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='event',
+            name='type',
+            field=models.CharField(choices=[('ASSIGNMENT', 'ASSIGNMENT'), ('EXAM', 'EXAM')], max_length=500),
```

Comment:
This will give us an error if there is already a practice event in the database. An idea is to change them to assignments before this database change.
Make sure to test this properly by adding a practise event and running the migration

Reply:
Tested with a practice event locally and it was changed to assignment on the migration.